msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase833511.java

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.broker;
+
+import static com.microsoft.identity.client.ui.automation.utils.CommonUtils.FIND_UI_ELEMENT_TIMEOUT;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiSelector;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost;
+import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator;
+import com.microsoft.identity.client.ui.automation.utils.UiAutomatorUtils;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+@SupportedBrokers(brokers = {BrokerMicrosoftAuthenticator.class})
+public class TestCase833511 extends AbstractMsalBrokerTest{
+
+    @Test
+    public void test_833511() {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        //perform device registration
+        mBroker.performSharedDeviceRegistrationDontValidate(username, password);
+
+        final UiDevice device =
+                UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+
+        final UiSelector sharedDeviceConfirmationSelector = new UiSelector()
+                .descriptionContains("Shared Device Mode")
+                .className("android.widget.ImageView");
+
+        //confirm that we are not in Shared Device Mode inside Authenticator - WPJ should fail
+        final UiObject sharedDeviceConfirmation = device.findObject(sharedDeviceConfirmationSelector);
```

Comment:
Please add an assertion that we see the correct error message.

Reply:
How does this "User is not authorized for registration" error appear in authenticator?

Reply:
"Shared device registration failed. Contact your admin if..."

Reply:
Is there an existing test case that asserts on a pop up message (the pop up clears after a while)? - I ran my assertion on the current screen not a pop up.

Reply:
@iamgusain Pls see comment above.

Reply:
I'm asserting that we are not in shared device mode - which I believe should equate to device registration failed, is this sufficient?

Reply:
it would be nice if we can also verify the error message that user sees, to catch any regressions there.

Reply:
Having said that based on Fadi's comment above it is probably not possible or is very hard to verify it using current UI Automation tools. In that case I am fine with the current verification but let's make sure that we add a new testcase (instrumented, Unit Test or even manual) to verify the error message so that it doesn't get missed.

Reply:
Noted

Reply:
If testing it like this is going to require the creation of another manual test case, it may be better to just keep this one manual, we wouldn't be getting much out of automating if this is the case. Although, if we can confirm this error message with component/unit testing, I think that is a good plan and we can keep this one UI automated with the current verification.